packages/components-css/icon-css/src/icon.scss

```diff
@@ -0,0 +1,13 @@
+@use "mixin";
+
+.nl-icon {
+  @include mixin.nl-icon;
+}
+
+.nl-icon > svg {
+  @include mixin.nl-icon__svg;
+}
+
+.nl-icon[dir="rtl"] {
```

Comment:
Ik zou gaan voor `:dir()` in plaats van `[dir]` https://caniuse.com/css-dir-pseudo

- Baseline support is er bijna
- Het is niet zo'n essentiële feature dat het Baseline support blocking zou moeten zijn
- Werkt ook als `dir` op een ancestor element zit, wat 99% van de gevallen zo zal zijn